### PR TITLE
fix(storage): upload works in browser

### PR DESCRIPTION
**Pull request by microchipgnu**

### CODE REVIEW

**Comment by SurgeCode** [comment]
There's an integration test for uploading in the testing package that we didn't complete if you feel like it, not urgent but if you're feeling it

**Review by cif — COMMENTED** [review]
Let's at a minimum, document these methods in README with examples. Even better, create an example in packages/app that does the file upload and use that working code as the example in docs

**Comment by microchipgnu** [comment]
@cif @SurgeCode added documentation and an example.

File: packages/storage/src/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+[//]: # `{ "title": "upload", "order": 0.1 }`
+
+# Upload in the Browser
+
+It's possible to use [File](https://developer.mozilla.org/en-US/docs/Web/API/File) to upload a file. 
+
+
+Example usage of storage method in a hypothetical React component:
+{% code title="StorageUI.ts" overflow="wrap" lineNumbers="true" %}
+
```
Comment (SurgeCode) [inline packages/storage/src/README.md:10]: Maybe adding a bit of a description here of what the method is doing and what it returns would be cool for people who don't know what is happening.

Comment (microchipgnu) [reply packages/storage/src/README.md:10]: That's good will do that.

**Review by sainthiago — APPROVED** [review]

**Review by SurgeCode — APPROVED** [review]
Looks good! 👍

**Comment by cif** [comment]
This should address and close https://github.com/Mintbase/mintbase-js/issues/222

**Pull request merged**